### PR TITLE
Follow-up to 1708

### PR DESCRIPTION
**Pull request by bermudezmt**
Signed-off-by: bermudezmt <bermudez.mt@gmail.com>

### Proposed changes

A few follow-up edits, and have started editing the Architecture & Concepts overview, with plans to add a diagram later.

### Related issues (optional)

Relates to #1322

### CODE REVIEW

**Comment by bermudezmt** [comment]
@mikhailshilkov I can't tag you for a review somehow but can you please have a look?

File: content/docs/intro/vs/cloud_template_transpilers.md [hunk 1]
```diff
@@ -8,65 +8,65 @@ menu:
...
-  [any SAML/SSO 2.0 provider]({{< relref "/docs/guides/saml" >}}), such as Azure Active Directory, Google G Suite,
-  or Okta, for auditing and access controls using your existing enterprise systems of record. AWS CloudFormation can
+* Pulumi integrates with your identity provider---including GitHub, GitLab, Atlassian, or
+  [any SAML/SSO 2.0 provider]({{< relref "/docs/guides/saml" >}}) (such as Azure Active Directory, Google G Suite,
+  or Okta)---for auditing and access controls using your existing enterprise systems of record. AWS CloudFormation can
   be manually integrated with those systems with greater effort.
 
-* Pulumi can be [use custom state management]({{< relref "/docs/intro/concepts/state#self-managed-backend" >}}),
-  and offers a self-hosting option, for greater control, including "behind the firewall" on-premises and hybrid
+* Pulumi can use [custom state management]({{< relref "/docs/intro/concepts/state#self-managed-backend" >}})
```
Comment (joeduffy) [inline content/docs/intro/vs/cloud_template_transpilers.md]: Thank you for catching this!

**Comment by joeduffy** [comment]
Looks great! 👍

File: content/docs/intro/concepts/_index.md [hunk 2]
```diff
@@ -8,24 +8,22 @@ menu:
...
+At the center of Pulumi is an open-source cloud object model, coupled with an evaluation runtime that knows how to take programs written in any language, understand the cloud resources necessary to execute them, and then plan and manage those resources in a robust way. This cloud runtime and object model is inherently language- and cloud-neutral, enabling Pulumi to support many languages and clouds rapidly.
 
-Each Pulumi [project]({{< relref "project.md" >}}) contains a [program]({{< relref "programming-model.md" >}}) --  executable code in a familiar language such as JavaScript, TypeScript, Python, or Go.  You will manage this program as you would any other source code, for example, using Git.
+Each Pulumi [project]({{< relref "project.md" >}}) contains a [program]({{< relref "programming-model.md" >}})---executable code in a familiar language such as JavaScript, TypeScript, Python, or Go. You will manage this program as you would any other source code. For example, by using Git.
```
Comment (mikhailshilkov) [inline content/docs/intro/concepts/_index.md]: Is that triple dash how we make the em-dash?

File: content/docs/intro/vs/cloud_template_transpilers.md (hunk 1 above)
Comment (bermudezmt) [reply content/docs/intro/vs/cloud_template_transpilers.md]: It was @mikhailshilkov who caught it. ;)

File: content/docs/intro/vs/cloud_template_transpilers.md [hunk 3]
```diff
@@ -8,65 +8,65 @@ menu:
...
 to the closed source AWS CloudFormation servers to provision infrastructure on AWS in the usual ways.
 
 ## Summary of Differences
 
 The transpiler approach gives you some of the benefits of Pulumi, with the following caveats:
 
-* Troposphere/CDK only support the AWS platform. Pulumi supports many clouds, including: major cloud platforms
-  such as Microsoft Azure, Google Cloud Platform, Kubernetes, and DigitalOcean; on-premises and hybrid technologies
-  such as VMWare vSphere and OpenStack; and online SaaS offerings like Cloudflare, Datadog, New Relic, and more.
+* Troposphere/CDK only supports the AWS platform. Pulumi supports many clouds, including _major cloud platforms_
```
Comment (mikhailshilkov) [inline content/docs/intro/vs/cloud_template_transpilers.md]: Troposphere and CDK are separate entities, is "supports" still correct? I would write "Troposphere and CDK only support"

File: content/docs/intro/concepts/_index.md (hunk 2 above)
Comment (bermudezmt) [reply content/docs/intro/concepts/_index.md]: Yes, that's how. Two dashes make an en dash.

File: content/docs/intro/vs/cloud_template_transpilers.md (hunk 3 above)
Comment (bermudezmt) [reply content/docs/intro/vs/cloud_template_transpilers.md]: Yeah, I went back and forth on this. Sorry for the premature merge. I'll include this change on the next PR.

**Pull request merged**